tests/foundry/client/data/documents/playlist.test-d.ts

```diff
@@ -0,0 +1,26 @@
+import { expectTypeOf } from "vitest";
+
+// @ts-expect-error - A Playlist requires name.
+new Playlist();
+
+// @ts-expect-error - A Playlist requires name.
+new Playlist({});
+
+const playlist = new Playlist({ name: "Some Playlist" });
+expectTypeOf(playlist).toEqualTypeOf<Playlist>();
+
+const playlistSound = new PlaylistSound({ name: "Some PlaylistSound" });
+
+expectTypeOf(playlist.playbackOrder).toEqualTypeOf<string[]>();
+expectTypeOf(playlist.visible).toEqualTypeOf<boolean>();
+expectTypeOf(playlist.prepareDerivedData()).toEqualTypeOf<void>();
+expectTypeOf(playlist.playAll()).toEqualTypeOf<Promise<Playlist | undefined>>();
+expectTypeOf(playlist.playNext()).toEqualTypeOf<Promise<Playlist | undefined | null>>();
+expectTypeOf(playlist.playSound(playlistSound)).toEqualTypeOf<Promise<Playlist | undefined>>();
+expectTypeOf(playlist.stopSound(playlistSound)).toEqualTypeOf<Promise<Playlist | undefined>>();
+expectTypeOf(playlist.stopAll()).toEqualTypeOf<Promise<Playlist | undefined>>();
+expectTypeOf(playlist.cycleMode()).toEqualTypeOf<Promise<Playlist | undefined>>();
+expectTypeOf(playlist.toAnchor()).toEqualTypeOf<HTMLAnchorElement>();
+
+// TODO: Modify to Playlist | null once data can be grabbed from CONFIG
```

Comment:
Do you know what this comment is about? It sounds like you copied at least portions of this over. Would you know what it's referring to?

I think this PR looks good, I just want to make sure whatever this comment is referring to isn't lost.

Reply:
I'm not sure - it existed on a couple other documents.

Reply:
I think this is referring to a now-dead idea that `ClientDocument#sheet` should return the default sheet; we can safely remove all of these style comments.

(The idea is dead because there's so much potential to remove and people should be using `instanceof` checks if they want to use specific sheet features)

Reply:
That's my kind of PR :)

Reply:
I can go clean those up and add the ApplicationV2 checks in those tests.  I'll do it after this batch all gets merged, as that will make it simpler to grab the ones I added.